lib/fastlane/plugin/wpmreleasetoolkit/helper/android/android_localize_helper.rb

```diff
@@ -299,6 +299,7 @@ def self.merge_xml_documents(all_xmls)
         #
         def self.apply_substitutions(tag)
           tag.content = tag.content.gsub('...', '…')
+          tag.content = tag.content.gsub('-', '&#8211;')
```

Comment:
@JavonDavis Wait, why was this change added this change? I remember explaining in #229 [why this was a bad idea](https://github.com/wordpress-mobile/release-toolkit/pull/229#issuecomment-820180415) 😓

This kind of misuse of the proper dash in translations (use of regular dash when en dash is expected for a range, use of en dash when regular dash is expected for hyphenation and other cases, …) have to be solved on case by case basis in GlotPress directly, either by the translators or by ourselves (it's an easy enough change that we don't need to know the language to edit the GlotPress entry), but definitively not automatically and unconditionally for all cases, because the right dash to use depends on the use case and typographical context.

_Besides, even if we wanted to do that, the substitution is incorrect here because Nokogiri (the XML gem) will treat that as verbatim text (you're telling it to replace the node `content` after all) so the `&` character followed by `#` followed by `8211` followed by `;`. Which will be encoded to `&amp;#8211;` once serialized in XML, as should be…_

Reply:
Hey @AliSoftware ! You're right!, thanks for catching this 🤦 !